### ISSUE TITLE
Notification email for the "Cancel" action to "Group email", the email body will be broken

### ISSUE THREAD

**Issue by kanalhui** [umbraco/Umbraco.Workflow.Issues#87]
When set the workflow with group email, the notification email of the "Cancel" is broken
![image](https://github.com/user-attachments/assets/fd03e777-3d4c-4078-8dc4-1a7c6998c6e3)

![image](https://github.com/user-attachments/assets/f6a0128c-6d98-4eb6-b030-e5f14d2f4555)

**Comment by nathanwoulfe** [comment]
Hey @kanalhui I suspect this is because the group doesn't have a language assigned - this is required to handle localizing the email body. Give that a try and let me know.

Ideally both those fields would be mandatory if either has a value.

**Comment by kanalhui** [comment]
Hi @nathanwoulfe

Thanks for your support. After we assign the language as English, it is working now. 
However, we tried to assign other language(our testing is using zh-hant ), the email body will missing the translation.
May I know is it required to define the translation ourself via "Translation"?

![image](https://github.com/user-attachments/assets/2f42711f-c411-4c27-8dbb-6554d80a236d)

**Comment by nathanwoulfe** [comment]
@kanalhui that is correct - the email body will only be translated if a translation dictionary exists. This is an XML file shipped as part of Workflow (or as a custom backoffice file), not translations maintained in the Translations section).

https://docs.umbraco.com/umbraco-cms/13.latest/extending/language-files shows how this works for 13 (it's very similar for 14).
